tests/ReflectionClassTest.php

```diff
@@ -290,4 +300,58 @@ protected function getGettersToCheck()
 
         return $allNameGetters;
     }
+
+    /**
+     * Returns list of ReflectionProperty that be checked
+     *
+     * @return array
+     */
+    protected function getPropertiesToCheck()
+    {
+        $names = [
+            'name',
+        ];
+
+        return $names;
+    }
+
+    /**
+     * @param string[] $membersToCheck
+     * @return array
+     */
+    protected function caseProvider($membersToCheck)
```

Comment:
I see you've moved method down, but because of that it's hard to see what you've changed in the method itself.